### PR TITLE
Dec 2019/Jan 2020 release

### PR DESCRIPTION
**Pull request by LCCarmody**
Added new terms to food intake avoidance, medical action avoidance, nutrition supplementation.  
Added new hydrotherapy branch, ablation therapy branch

### CODE REVIEW

File: maxo-base.obo [hunk 1]
```diff
@@ -753,55 +756,13 @@ is_a: MAXO:0000009 ! nutrition intervention
 property_value: http://purl.org/dc/elements/1.1/date 2019-01-19T21:01:21Z xsd:dateTime
 created_by: http://orcid.org/0000-0001-7941-2961
 
-[Term]
-id: MAXO:0000099
-name: food consumption avoidance
-is_a: MAXO:0000056 ! therapeutic nutritional avoidance
-property_value: http://purl.org/dc/elements/1.1/date 2019-01-19T21:01:41Z xsd:dateTime
-created_by: http://orcid.org/0000-0001-7941-2961
-
```
Comment (matentzn) [inline maxo-base.obo]: The following food avoidance terms are just lost - not even obsoleted. Is that intended?

Comment (matentzn) [reply maxo-base.obo]: You cant see this here in the snipped, but if you look at the diff, go to maxo-base.obo, click show diff and browse doen to the avoidance terms, they appear gone. is this only in base? or also maxo.owl?

Comment (LCCarmody) [reply maxo-base.obo]: I'm confused. It is there, but now called 'nutrition intake avoidance.  Maybe I forgot to just add this as a synonym?

Comment (LCCarmody) [reply maxo-base.obo]: Oh, I see. This was a label and not a synonym.  Hmmm... would making a "therapeutic nutritional avoidance" fix it?

**Comment by matentzn** [comment]
Forget what I said! All good to go! Thanks! (I found now what my mistake was, everything is correct in maxo.obo)

**Pull request merged**